src/frontend/templates/index_full_chat.html

```
@@ -44,7 +44,7 @@ <h1>CHANAKYA</h1>
             <button id="recordButton" title="Manual Record">🎤</button>
             <button id="playResponseButton" title="Play Last Response">🔊</button>
             <button id="callModeButton" title="Start/End Voice Call">📞</button>
-            <button id="keywordListenToggleButton" title="Toggle Keyword Listening">👂</button> 
+            <button id="keywordListenToggleButton" title="Toggle Keyword Listening">👂</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)